mipengine/tests/node/test_udf_generator.py

```diff
@@ -348,11 +344,11 @@ def udf_many_params(
 @pytest.mark.parametrize("test_input,expected", test_cases_generate_udf)
 def test_generate_udf(test_input, expected):
     assert (
-        generate_udf(
-            func_name=test_input["func_name"],
-            udf_name=test_input["udf_name"],
-            positional_args=test_input["positional_args"],
-            keyword_args=test_input["keyword_args"],
-        )
-        == expected
+            generate_udf(
+                func_name=test_input["func_name"],
+                udf_name=test_input["udf_name"],
+                positional_args=test_input["positional_args"],
+                keyword_args=test_input["keyword_args"],
+            )
+            == expected
     )
```

Comment:
Did you change the indentation? We need to all use the same formatter -> [black](https://github.com/psf/black)